### PR TITLE
Impl zpopmin command

### PR DESCRIPTION
**Pull request by kommradHomer**
implemented zpopmin command.
written a test method

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -191,6 +191,28 @@
 				    </execution>
 				</executions>		
 			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
```
Comment (sazzad16) [inline pom.xml]: Why this?

Comment (kommradHomer) [reply pom.xml]: Oh. I thought I removed that one. I'll remove that and commit again

**Comment by kommradHomer** [comment]
removed the unnecessary plugin

File: pom.xml [hunk 2]
```diff
@@ -191,6 +191,7 @@
 				    </execution>
 				</executions>		
 			</plugin>
+
```
Comment (sazzad16) [inline pom.xml]: Remove this.

**Comment by gkorland** [comment]
what about JedisClusterCommands?

**Comment by SuShuilian** [comment]
I am looking forward to seeing this new command available in release version.
When will it be available in release version, any plan?

**Pull request merged**